src/trans/gpu/external/setup_trans.F90

```diff
@@ -98,7 +98,7 @@ SUBROUTINE SETUP_TRANS(KSMAX,KDGL,KDLON,KLOEN,LDSPLIT,PSTRET,&
 !        R. El Khatib 07-Mar-2016 Better flexibility for Legendre polynomials computation in stretched mode
 !     ------------------------------------------------------------------
 
-USE PARKIND1,        ONLY: JPIM, JPRB, JPRD
+USE PARKIND1,        ONLY: JPIM, JPRB, JPRD, JPIB
```

Comment:
I don't think you need this do you?
```suggestion
USE PARKIND1,        ONLY: JPIM, JPRB, JPRD
```

Reply:
You are right, this is a leftover from before we moved to the right C_SIZEOF. I like that you always keep an eye on unnecessary "use" statements!